frontend/src/components/Navbar.tsx

```diff
@@ -27,14 +27,31 @@ const Navbar = () => {
         <Button color="inherit" component={Link} to="/chains/new-signup">
           {t("startNewChain")}
         </Button>
-        { user ?
+        {user ? (
           <Button color="inherit" component={Link} to="/users/logout">
             {t("logout")}
-          </Button> :
+          </Button>
+        ) : (
           <Button color="inherit" component={Link} to="/users/login">
             {t("login")}
-          </Button> 
-        }
+          </Button>
+        )}
+
+        {userData != null && userData.role === "admin" ? (
+          <Button color="inherit" component={Link} to="/chains">
+            {t("admin")}
+          </Button>
+        ) : null}
+
+        {userData != null && userData.role === "chainAdmin" ? (
```

Comment:
You can remove the null check with null coalescion operator: userData?.role === "admin"